src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -370,6 +434,90 @@ TxQ::eraseAndAdvance(TxQ::FeeMultiSet::const_iterator_type candidateIter)
 
 }
 
+auto
+TxQ::erase(TxQ::TxQAccount& txQAccount,
+    TxQ::TxQAccount::TxMap::const_iterator begin,
+        TxQ::TxQAccount::TxMap::const_iterator end)
+            -> TxQAccount::TxMap::iterator
+{
+    for (auto it = begin; it != end; ++it)
+    {
+        byFee_.erase(byFee_.iterator_to(it->second));
```

Comment:
Nevermind: I see that `byFee_` is a `multiset`. Just using `erase` on the value is wrong.